content/en/synthetics/ci.md

```diff
@@ -0,0 +1,330 @@
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+<div class="alert alert-warning">
+This feature is in private beta.
+</div>
+
+On top of executing your tests at predefined intervals, you can also execute Datadog Synthetics tests on-demand using the dedicated API endpoints. You can execute Datadog Synthetics tests in your continuous integration (CI) pipelines, enabling you to block the deployment of branches which would break your key features and endpoints.
+
+This function allows you to avoid spending time fixing issues on production, and to catch bugs and regressions earlier in the process.
+
+On top of these API endpoints, Datadog provides and maintains a command line interface (CLI), allowing you to easily integrate Datadog Synthetics with your CI tooling.
+
+## API usage
+
+The trigger endpoint provides the list of triggered checks alongside their result identifiers. A polling endpoint is available to obtain the full results of the tests if they are available.
+
+
+### Trigger tests endpoint
+
+The test triggering endpoint supports starting up to 50 tests in one request.
+
+* **Endpoint**: `https://api.datadoghq.com/api/v1/synthetics/tests/trigger/ci`
+* **Method**: `POST`
+* **Argument**: A JSON object containing the list of all tests to trigger and their configuration override.
+
+#### Request data structure
+
+```json
+{
+    "tests": [TEST_TO_TRIGGER, TEST_TO_TRIGGER, ...]
+}
+```
+
+The `TEST_TO_TRIGGER` objects are composed of the `public_id` (required) of the test to trigger and optional configuration overrides ([see below][#adding-and-running-tests] for description of each field).
+
+The public identifier of a test can be either the identifier of the test found in the URL of a test details page (for `https://app.datadoghq.com/synthetics/details/abc-def-ghi`, it would be `abc-def-ghi`) or the full URL to the details page (that is, `https://app.datadoghq.com/synthetics/details/abc-def-ghi`).
+
+#### Example request
+
+```bash
+curl -X POST \
+-H 'Content-Type: application/json' \
+-H "DD-API-KEY: ${api_key}" \
+-H "DD-APPLICATION-KEY: ${app_key}" \
+-d '{
+    "tests": [
+        {
+            "public_id": "abc-def-ghi",
+            "allowInsecureCertificates": true,
+            "basicAuth": { "username": "test", "password": "test" },
+            "body": "{\"fakeContent\":true}",
+            "bodyType": "application/json",
+            "cookies": "name1=value1;name2=value2;",
+            "deviceIds": ["laptop_large"],
+            "followRedirects": true,
+            "headers": { "NEW_HEADER": "NEW VALUE" },
+            "locations": ["aws:us-east-1"],
```

Comment:
We realized us-east-1 wasn't one of our locations, let's change it to one of our actual locations
```suggestion
            "locations": ["aws:us-west-1"],
```